SIT.Manager/Models/Aki/AkiCharacter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SIT.Manager.Models.Aki;
+public class AkiCharacter(AkiServer parent, string username, string password) : ObservableObject
+{
+#pragma warning disable CA1507 // Use nameof to express symbol names
+    [JsonProperty("Username")]
+    public string Username { get; init; } = username;
+    [JsonProperty("Password")]
+    public string Password { get; init; } = password;
+    [JsonProperty("ProfileID")]
+    public string ProfileID { get; internal set; } = "";
+    [JsonProperty("Edition")]
+#pragma warning restore CA1507 // Use nameof to express symbol names
```

Comment:
Do these JsonProperty names need the capitalization? As the other models don't use them see SIT.Manager/Models/Aki/AkiMiniProfile.cs or in the case if SIT.Manager/Models/Aki/AkiServer.cs you could just use nameof.

Either way would be nicer to not have to use ```#pragma warning disable CA1507```

Reply:
AkiMiniProfile uses lowercase because it needs to deserialize a server response. AkiServer idk why it went with nameof here honestly, My main concern with using nameof is that if we ever change the name of the variable then anything serialized with the previous variable name will now be incompatible which doesn't seem like a good design choice

Reply:
That's a fair point probably makes sense to not use nameof for these then